packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -1,40 +1,54 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-accordion-item-background-color: defines the background color for the component.
+ * @prop --calcite-accordion-item-border-color: defines the border color for the component.
```

Comment:
`--calcite-accordion-item-border-color` doesn't seem to be used in `accordion-item`'s styling.

Reply:
thanks for the catch!